app/models/default_report_service.rb

```diff
@@ -0,0 +1,38 @@
+# This class mimics external_report.rb
+class DefaultReportService
+  DefaultReportServiceAppID   = 'DEFAULT_REPORT_SERVICE_CLIENT'
+  DefaultReportDomainMatchers = '*.concord.org concord-consortium.github.io localhost'
+  ReportViewUrl = 'https://concord-consortium.github.io/portal-report/'
+
+
+  ReportTokenValidFor = 2.hours
+
+  def self.instance
+    @instance || self.new()
+  end
+
+  def reportViewUrl
+    ENV['REPORT_VIEW_URL'] || ReportViewUrl
```

Comment:
I'm not sure, but I think it would be better to move this default into the `app_environment_variables.sample.rb`.
That way we can consolidate the available environment variables that devs can use to customize the app.
Since it is likely that a dev might not update their `app_environment_variables.rb` with this new value, it would be useful for this code to throw an exception if the ENV variable is not set.